bundler/lib/bundler/fetcher.rb

```diff
@@ -4,6 +4,7 @@
 require "cgi"
 require "securerandom"
 require "zlib"
+require "rubygems"
```

Comment:
I don't think it's possible that this isn't loaded, is it? I could totally be wrong.

Reply:
it's not - there's a `require "rubygems/request"` right below, if nothing else. I'll yank it.